developer/frameworks/articles/configuration/02-creating-a-configuration-interface.markdown

```diff
@@ -81,9 +79,12 @@ This automatically scopes your configuration to `SYSTEM`.
 | described below, we recommend defaulting to the bnd annotations.
 
 | **Also Note:** Your project depends on the `-metatype: *` line in it's
-| `bnd.bnd` file. This line is generated automatically from your bundle's
-| `.build.properties` file, and is required to provide information about your
-| app's configuration options so that a configuration UI can be generated.
+| `MANIFEST.MF` file. If you're in a [Liferay
```

Comment:
@pei-jung I got this wrong. Inspecting an -api JAR file shows that this actually gets generated into a *Configuration.xml file in the module JAR's `OSGI-INF/metatype` folder. I'll update and push a commit.